config/test-openhpc.yml

```diff
@@ -1,21 +1,73 @@
 ---
```

Comment:
The script used to run this cluster, tests/test-openhpc.sh, applies this config file and `openhpc.yml`, so we only need to include variables that we want to change in that file. I suppose it would be helpful if this file had a comment to that effect.

Reply:
Gotcha!